### DISCUSSION TITLE
add new shaders and land-use properties

### DISCUSSION THREAD

**Discussion by s-martiz** [UMEP-dev/UMEP#573] [Q&A] [answered]
I have a question about the possibility of extending the use of UMEP:
1. Is it possible to model shading solutions other than trees (shades, courtyards, etc.)?
2. Can ground cover with different emissivity and albedo properties be added? It seems to me that Solweig uses correlations to evaluate the temperature difference between the air and the ground surface. What happens to these correlations when layers are added to the landcoverclasses_2016a.txt file?

Kind regards

**Comment by biglimp** [comment, marked as answer]
1. We use the concept of 2.5 dimensions. Implications of this make it not possible to model Tmrt and include balconies, shades etc. However, if you only want to look at shadow patterns with the Shadow Generation tool you can use the CDSM to represent 3D objects by setting transmissivity to 0.
2. You can add as many ground cover as you like. We use empirical relationships for surface temperature calculations so to include new ground covers you need to either make an educated guess or do field experiments. Heads up. We are working on a new surface temperature scheme which probably will make it easier to include new wall materials and ground covers.

**Comment by s-martiz** [comment]
Thank you for your feedback @biglimp. 

I am indeed interested in adding new ground covers. Do you have any information on the type of experimental set-up to be used? For the moment, I've taken most of the information directly from the code or from this article "SOLWEIG 1.0 - Modelling spatial variations of 3D radiant fluxes and mean radiant temperature in complex urban settings".
Thank you and well done for your work :)

**Reply by biglimp** [reply]
Have look in Lindberg, Onomura and GRimmond 2016, IJBM. THere you can see how the ground cover scheme was developed and how you should go about doing observation to inculde new ground covers.

**Reply by s-martiz** [reply]
Dear @biglimp 

I hope you don't mind me following up on a discussion we had some time ago. I am still interested in testing different landcover types with UMEP as it seems to be a common request from urban planners who are defining innovative materials to fight against urban heat islands. I have a question about the methodology used by Solweig. I read in the paper by Lindberg, Onomura, and Grimmond (2016, IJBM) the following sentence: "Measurements of surface temperatures for two different types of surfaces (short grass and dark-colored asphalt) were conducted to obtain relations between solar elevation and Tdiffmax for the Ts estimation of the surfaces." Is the surface temperature in Solweig estimated only based on the position of the sun? I am struggling to understand where albedo and emissivity come into play in estimating this temperature.

Best,
Simon

**Reply by biglimp** [reply]
The surface temperature scheme is, as you have noticed derived from empirical data. Incoming shortwave radiation (clearness) is also an important factor in the current Ts-scheme. The material characteristics is embedded into these data. Emissivity is used to convert to energy flux via Stefan Bolzman. Albedo and emissivity is disconnected, i.e. if you change albedo is does not alter the emissivity. Albedo is used to alter the outgoing shortwave radiation.
